packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -21,6 +23,7 @@ export default class EditorEnvelope extends PureComponent {
         editorRegistry: PropTypes.object.isRequired,
         validationErrors: PropTypes.array,
         onEnterKey: PropTypes.func,
+        helpMessage: PropTypes.string,
```

Comment:
Do we miss a default value here?

Reply:
I would say that `helpMessage` is `undefined` and if we have an `undefined` `helpMessage` we do not render the help-icon and title